summit.html

```diff
@@ -51,454 +49,11 @@ <h1 class="section-heading">The practicalities</h1>
-                    </p>
-                  </div>
-                </details>
-              </td>
-              <td><span class="tag bd-success" style="border: 1px solid blue !important;">Intermediate</span></td>
-              <td>14:30</td>
-            </tr>
-
-            <tr><td><i>Break</i></td><td></td><td>15:15</td></tr>
-
-            <tr>
-              <td>
-                <details>
-                  <summary>Q&amp;A</summary>
-                  <div>
-                    <p>
-                      Bring your own questions or ideas to this open
-                      session where you get a chance to stump the
-                      experts or get feedback.
-                    </p>
-                    <p>
-                      <b>Moderator:</b> Edvin Agnas (Volvo Group)
-                    </p>
-                    <p>
-                      <a href="https://youtu.be/ee7Wah4_QAI">Video</a>
-                    </p>
-                  </div>
-                </details>
-              </td>
-              <td><span class="tag bd-success">Beginner</span></td>
-              <td>15:45</td>
-            </tr>
-
-            <tr>
-              <td>
-                <details>
-                  <summary>Wrap up</summary>
-                  <div>
-                    <p>
-                      We wrap up and summarize the day.
-                    </p>
-                    <p>
-                      <b>Moderator:</b> Edvin Agnas (Volvo Group)
-                    </p>
-                  </div>
-                </details>
-              </td>
-              <td><span class="tag bd-success">Beginner</span></td>
-              <td>16:15</td>
-            </tr>
-
-            <tr><td><i>End of day 1</i></td><td></td><td>16:30</td></tr>
-
-            <tr>
-              <td>
-                <details>
-                  <summary>Eiffel Summit dinner</summary>
-                  <div>
-                    <p>
-                      We take the opportunity to socialize and grab
-                      something to eat. A table reservation has been
-                      made for those who indicated interest during
-                      registration and confirmed it by email to Marie
-                      Aubourg. We can't guarantee that the restaurant
-                      will be able to accommodate anyone else.
-                    </p>
-                    <p>
-                      <b>Location:</b>
-                      <a href="https://palatset.com">Biljardpalatset</a>,
-                      <a href="https://maps.app.goo.gl/JTrYPkLJk1m5KbQU8">
-                        Ekelundsgatan 11
-                      </a>
-                    </p>
-                  </div>
-                </details>
-              </td>
-              <td></td>
-              <td>18:00</td>
-            </tr>
-
-          </tbody>
-        </table>
+        TBD
 
         <h2>Day 2 (Thursday, September 5)</h2>
```

Comment:
```suggestion
        <h2>Day 2 (Wednesday, September 8)</h2>
```

Reply:
Yes, thanks! Fixed in [1f439fd](https://github.com/eiffel-community/eiffel-community.github.io/pull/76/commits/1f439fd348524fcc5af439a7f96edddf38479887)